bufferserver/src/main/java/com/datatorrent/bufferserver/server/Server.java

```diff
@@ -73,9 +73,12 @@
   private InetSocketAddress address;
   private final ExecutorService serverHelperExecutor;
   private final ExecutorService storageHelperExecutor;
+  private boolean backPressureEnabled;
 
   private byte[] authToken;
 
+  private static final boolean BACK_PRESSURE_ENABLED = !Boolean.getBoolean("BUFFERSERVER.BACK_PRESSURE_DISABLE");
```

Comment:
It could be useful in testing or other scenarios where we want to programmatically supply this or configure it per instance basis. Also, I don't think it would be right for DataList to refer to this as a static field on the Server, it should either be passed in or better it has a reference to Server. Also when going to the per-stream approach I would still think something like this would have to be passed in.